app/controllers/page_views_controller.rb

```diff
@@ -4,74 +4,31 @@ class PageViewsController < ApplicationMetalController
   include ActionController::Head
 
   def create
-    page_view_create_params = if session_current_user_id
-                                page_view_params.merge(user_id: session_current_user_id)
-                              else
-                                page_view_params.merge(counts_for_number_of_views: 10)
-                              end
-
-    PageView.create(page_view_create_params)
+    page_view_create_params = params.slice(:article_id, :referrer, :user_agent)
```

Comment:
I decided to get rid of the `page_view_params` method and inline param handling instead because this is the only place where we use it and like being able to see what exactly the params contain without having to scroll around.